### PR TITLE
features-pages-layout

### PR DESCRIPTION
**Pull request by Alqanar**
~~Work in progress~~

## Update 5/6/22 (Delia):
This PR adds a `features` layout and updates files in the `/features` directory to implement that layout. This ports the final pages from `/web` to `/next`.

### Updates these pages to use the `features` layout:
- [/features/access-requests](https://teleport-docs-next-git-features-pages-layout-gravitational.vercel.app/features/access-requests/)
- [/features/multi-factor-auth/](https://teleport-docs-next-git-features-pages-layout-gravitational.vercel.app/features/multi-factor-auth/)

### Adds these pages:
- [/features/rbac-for-ssh/](https://teleport-docs-next-git-features-pages-layout-gravitational.vercel.app/features/rbac-for-ssh/)
- [/features/sso-for-ssh/](https://teleport-docs-next-git-features-pages-layout-gravitational.vercel.app/features/sso-for-ssh/)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202111964751622

### CODE REVIEW

**Comment by C-STYR** [comment]
@Alqanar we should be getting the content for these pages in the next week or so - please follow this ticket: https://app.asana.com/0/1201922182803749/1201999850180493/f

**Comment by deliaconstantino** [comment]
Previews:
- [/features/access-requests](https://teleport-docs-next-git-features-pages-layout-gravitational.vercel.app/features/access-requests/)
- [/features/multi-factor-auth/](https://teleport-docs-next-git-features-pages-layout-gravitational.vercel.app/features/multi-factor-auth/)
- [/features/rbac-for-ssh/](https://teleport-docs-next-git-features-pages-layout-gravitational.vercel.app/features/rbac-for-ssh/)
- [/features/sso-for-ssh/](https://teleport-docs-next-git-features-pages-layout-gravitational.vercel.app/features/sso-for-ssh/)

File: pages/features/access-requests/index.mdx [hunk 1]
```diff
@@ -83,93 +66,110 @@ import TryTeleport from "components/TryTeleport";
...
-        </p>
-        <p>
-          Approvals do not have to be done manually. Implement your own access approval {' '}
-          logic with Teleport API using your favorite programming language.
-        </p>
-      </>
-    }>
-  <Box boxShadow="0 4px 32px rgba(0, 0, 0, 0.24)" borderRadius="md" lineHeight="0">
+
+{" "}
```
Comment (sandylcruz) [inline pages/features/access-requests/index.mdx:94]: You can remove this

**Comment by sandylcruz** [comment]
Overall looks good, just add `<LiveChatLoaderProvider>` around `<Drift />` to align with the other layouts. Also, please update the `alt` text to not just be a reiteration of the title of the page, but a description of what the image is (for screen readers). [Here's](https://moz.com/learn/seo/alt-text) more information on writing `alt` text.

File: pages/features/access-requests/index.mdx (hunk 1 above)
Comment (deliaconstantino) [reply pages/features/access-requests/index.mdx]: removed

**Comment by deliaconstantino** [comment]
@sandylcruz thanks. Added in the `<LiveChatLoaderProvider>` for `<Drift />` and updated alt tags throughout this PR.

**Review by sandylcruz — APPROVED** [review]

**Pull request merged**